impeller/aiks/paint_pass_delegate.cc

```diff
@@ -39,6 +39,7 @@ std::shared_ptr<Contents> PaintPassDelegate::CreateContentsForSubpassTarget(
   contents->SetTexture(target);
   contents->SetSourceRect(Rect::MakeSize(target->GetSize()));
   contents->SetOpacity(paint_.color.alpha);
+  contents->SetDeferApplyingOpacity(true);
 
   return paint_.WithFilters(std::move(contents), false, effect_transform);
```

Comment:
Since we'll have to diverge with the ordering of `WithFilters` at some point anyhow, maybe we should just do the `WithFilters` logic inline here and build the filter chain exactly as it needs to be. Then, we can set the alpha absorption flag on the color filter here where it's needed instead of having to do it in the dispatcher or color filter procs.

To facilitate this, we could also add a `ColorFilterContents` class that extends `FilterContents` which is inherited by all of the color filters -- this would hold the setter and remove the repeated code without having to place it on filters that will never use it.

Reply:
Done. I adjusted the PR according to your suggestion, the one difference is that I added a method `Paint::WithFiltersForSubpassTarget`, which can reuse some logic.  Let me know what you think!